src/components/pages/Step4.vue

```diff
@@ -588,7 +606,8 @@ export default {
                   // subject_gender: this.gender,
                   settings_data_sharing: this.data_sharing,
                   settings_pose_model: this.pose_model,
-                  settings_framerate: this.framerate
+                  settings_framerate: this.framerate,
+                  sessionName: this.sessionName
```

Comment:
Let's follow the same conventions: `settings_session_name`

Reply:
Applied.